pkg/alertmanager/multitenant.go

```diff
@@ -721,10 +728,12 @@ func (am *MultitenantAlertmanager) computeConfig(cfgs alertspb.AlertConfigDescs)
 	// Grafana configuration.
 	case cfgs.Mimir.RawConfig == am.fallbackConfig:
 		level.Debug(am.logger).Log("msg", "mimir configuration is default, using grafana config with the default globals", "user", cfgs.Mimir.User)
+		am.multitenantMetrics.grafanaConfigSize.WithLabelValues(cfgs.Grafana.User).Set(float64(len(cfgs.Grafana.RawConfig)))
```

Comment:
We could do this once at the top of the method, before entering the `switch`. Even if the Grafana config is ignored, we'll get the size in bytes.

Reply:
Sure, I wasn't sure if we actually wanted to record it in those cases.
I've moved it out of `computeConfigs` and into `syncConfigs` instead, mainly because of the user reference from the `cfgMap` but also considering there are other metrics being recorded there. Please let me know if you agree 🙌